gsoc/gsoc2016/ideas_astropy.md

```diff
@@ -150,3 +150,30 @@ This project includes looking at other astronomy codes to see how they tackle th
 
 See https://github.com/OpenAstronomy/openastronomy.github.io/pull/27 for a discussion of some problems and possible solutions that will be addressed in this project.
 
+
+### Bridge sherpa and astropy fitting
+
+*Suggested Mentor(s):* D. Burke, T. Aldcroft, H. M. Guenther
```

Comment:
Doug is on vacation now and potentially out of contact.  I can be listed for primary for now and we'll see how things play out if we find a suitable student.